README.md

```diff
@@ -219,33 +208,50 @@ composes stream data instead of stream processors (functions).  A stream is
 just like a list and is explicitly passed around to functions that process the
 stream.  Therefore, no special operator is needed to join stages in a streaming
 pipeline, just the standard function application (`$`) or reverse function
-application (`&`) operator is enough.  Combinators are provided in
-`Streamly.Prelude` to transform or fold streams.
+application (`&`) operator is enough.
 
 ## Concurrent Stream Generation
 
-Monadic construction and generation functions e.g. `consM`, `unfoldrM`,
-`replicateM`, `repeatM`, `iterateM` and `fromFoldableM` etc. work concurrently
-when used with appropriate stream type combinator (e.g. `asyncly`, `aheadly` or
-`parallely`).
+`consM` or its operator form `|:` can be used to construct a stream from
+monadic actions. A stream constructed with `consM` can run the monadic actions
+in the stream concurrently when used with appropriate stream type combinator
+(e.g. `asyncly`, `aheadly` or `parallely`).
 
-The following code finishes in 3 seconds (6 seconds when serial):
+The following code finishes in 3 seconds (6 seconds when serial), note the
+order of elements in the resulting output, the outputs are consumed as soon as
```

Comment:
"the outputs are produced"?

Reply:
Or maybe consumed after all. Huh.

Reply:
"outputs are consumed as soon as they are produced" ;)